LEAF_Request_Portal/templates/view_reports.tpl

```diff
@@ -816,7 +816,8 @@ function showJSONendpoint() {
     delete query.limit;
     delete query.limitOffset;
     let queryString = JSON.stringify(query);
-    let jsonPath = pwd + leafSearch.getLeafFormQuery().getRootURL() + 'api/form/query/?q=' + queryString + '&x-filterData=recordID,'+ Object.keys(filterData).join(',');
+    let xFilterData = '&x-filterData=recordID,'+ Object.keys(filterData).join(',');
+    let jsonPath = pwd + leafSearch.getLeafFormQuery().getRootURL() + 'api/form/query/?q=' + queryString + xFilterData;
```

Comment:
Just a quick check.  Unsure whether I missed this before or just don't remember the answer.  Is pwd accessing the client computer or the server?  If server, it looks like we're sending infrastructure information to the client?  Something that's a no-no.

Reply:
It looks like it is information that would be present to the client. 

    let pwd = document.URL.substr(0,document.URL.lastIndexOf('?'));